src/Simplic.FileStructure.Service/FileStructureDocumentPathService.cs

```diff
@@ -137,5 +137,21 @@ public IEnumerable<FileStructureDocumenPath> GetByDocumentId(Guid documentId)
         {
             return repository.GetByDocumentId(documentId);
         }
+
+        public IEnumerable<FileStructureDocumenPath> GetByDirectoryId(Guid directoryId)
+        {
+            return repository.GetByDirectoryId(directoryId);
+        }
+
+        
+
+        public bool IsProtected(List<Guid> guids, SeperatedStringBuilder ssb = null)

```

Comment:
Use a local variable instead of an argument here: `SeperatedStringBuilder ssb = null`...
Add comments